<ui version="4.0" >
 <class>ResultDialog</class>
 <widget class="QDialog" name="ResultDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Result</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="resultLabel" >
     <property name="text" >
      <string>You Win !</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Time :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeLabel" >
       <property name="text" >
        <string>00h 00m 00s</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="restartButton" >
     <property name="text" >
      <string>Restart game</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="newButton" >
     <property name="text" >
      <string>New game</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
